system/metis/Chart.yaml

```
@@ -1,12 +1,12 @@
 apiVersion: v2
 name: metis
-version: 1.3.0
+version: 1.4.0
 description: Read-only DB for replicas of OpenStack DBs and Project Masterdata
 dependencies:
   - condition: mariadb.enabled
     name: mariadb
     repository: oci://keppel.eu-de-1.cloud.sap/ccloud-helm
-    version: 0.14.3
+    version: 0.15.2
```

```suggestion
    version: 0.15.3
```

Versions 0.15.0-0.15.2 contain a bug in service selector

---

Thank you for the hint 💯